shared/src/packages/orders/libs/types/order-find-by-request-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type OrderFindByRequestDto = {
+  'userId': string;
+  'businessId': string;
+  'driverId': string;
+};
```

Comment:
why props in single quotes? why strings?

Reply:
fixed it, it was a bad idea)